### PR TITLE
[#2886] improvement(ci): optimize  Docker container to suite CI framework

### PR DESCRIPTION
**Pull request by zhoukangcn**
### What changes were proposed in this pull request?

- remove log from container stdout
- improvement: add config to avoid storage_medium_check

### Why are the changes needed?

Fix: #2886

### Does this PR introduce _any_ user-facing change?

N/A

### How was this patch tested?

CI

### CODE REVIEW

File: dev/docker/doris/start.sh [hunk 1]
```diff
@@ -37,6 +36,7 @@ PRIORITY_NETWORKS=$(echo "${CONTAINER_IP}" | awk -F '.' '{print$1"."$2"."$3".0/2
 echo "add priority_networks = ${PRIORITY_NETWORKS} to fe.conf & be.conf"
 echo "priority_networks = ${PRIORITY_NETWORKS}" >> ${DORIS_FE_HOME}/conf/fe.conf
 echo "priority_networks = ${PRIORITY_NETWORKS}" >> ${DORIS_BE_HOME}/conf/be.conf
+echo "disable_storage_medium_check = true" >> ${DORIS_FE_HOME}/conf/fe.conf
```
Comment (yuqi1129) [inline dev/docker/doris/start.sh:39]: Will this configuration prevent the check for left storage when starting?

Comment (zhoukangcn) [reply dev/docker/doris/start.sh:39]: Yes, It's no need to check this. 

> If disable_storage_medium_check is true, ReportHandler would not check tablet's storage medium and disable storage cool down function, the default value is false. You can set the value true when you don't care what the storage medium of the tablet is.

When we have SSD and HDD on same machine, this check is helpful.

When we use in CI, this config may cause BE refuse request, and finally make `create table` failed probabilistic (when the container has just been started)

I am checking it in CI

**Review by yuqi1129 — APPROVED** [review]
LGTM

**Comment by yuqi1129** [comment]
@zhoukangcn 
Could you change the image version to `0.1.3` by the way, I have verified it.

**Comment by zhoukangcn** [comment]
> @zhoukangcn 
> Could you change the image version to `0.1.3` by the way, I have verified it. 

OK，I can do it later

**Comment by zhoukangcn** [comment]
@yuqi1129 done. Please take a look when you have time.

**Comment by zhoukangcn** [comment]
@yuqi1129 please review PR #2922

**Review by yuqi1129 — APPROVED** [review]
LGTM

**Pull request merged**